meta.yaml

```diff
@@ -1,7 +1,7 @@
 # For conda build
 package:
   name: rmg
-  version: "0" # set by build.sh, which gets from rmgpy/__init__.py
+  version: 2.1.9
```

Comment:
could use `version: {{ environ.get('GIT_DESCRIBE_TAG', '') }}` once you've set the tag

Reply:
That's true, I'll change it. It didn't work during testing because the tags aren't on this branch, but it should work in general.